specs/release-automation/user-triggered-automation-workflow.puml

```diff
@@ -0,0 +1,63 @@
+@startuml
+title Release Automation Process
+
+start
+group RA-1: Trigerring the Process
+    :User passes link to excel dictionary on OSF to use for the release;
+    :User passes OSF personal access token;
+    :User triggers release process;
+end group
+
+group RA-2: Creating the Release Files
+    :Retrieve Excel Dictionary;
+    :Parse metadata from files sheet;
+    :Extract file name and type;
+    :Extract dictionary version from the dictionary file name;
+    :Extract dictionary version from the dictionary summary sheet;
+    if (Version missmatch) then(yes)
+        :Issue error;
+        stop
+    else (no)
+        :Parse the name template string;
+        :Construct file names using populated template and file type;
+        :Extract the part value from sheets for this file;
+        :Check parts sheet for the partID;
+        if (Part is a set) then (yes)
```

Comment:
How do you know if a partID is a set from the parts sheet? Isn't it better to check the sets sheet?